README.md

```diff
@@ -1,7 +1,7 @@
 # lamden-js
-This is the Lamden javascript implementation used in the [Lamden Wallet Chrome Extention](https://chrome.google.com/webstore/detail/lamden-wallet-browser-ext/lgkgmnhecgdjiifepobmobkeeeheakko) ([Github](https://github.com/Lamden/wallet)).
+This is the Lamden javascript implementation used in the [Lamden Wallet Chrome Extention](https://chrome.google.com/webstore/detail/lamden-wallet-browser-ext/fhfffofbcgbjjojdnpcfompojdjjhdim) ([Github](https://github.com/Lamden/wallet)).
 
-This package should work in Node.js and Broswer implementations.
+This package should work in Node.js and Browswer implementations.
```

Comment:
should this read 'Browser' ?

Reply:
I see you fixed this in a later commit, please ignore above.

Reply:
Should be Browser